schema/ownership-or-control-statement.json

```diff
@@ -20,6 +20,19 @@
     "statementDate": {
       "$ref": "components.json#/definitions/StatementDate"
     },
+    "componentOfStatement": {
+      "title": "Component of statement",
+      "description": "Is this statement a component of another statement? A `ownershipOrControlStatement` where `componentOfStatement` is 'false' is assumed to be the primary ownership-or-control declaration and to be a complete representation of the declarable ownership relationship between the `subject` and the `interestedParty`. A statement where `componentOfStatement` is 'true' MUST be the component of an `ownershipOrControlStatement` where `componentOfStatement` is `false` and come before that statement in a BODS package or stream. A component statement MAY be a partial representation of ownership between two parties, representing the portion relevant to the generating statement.",
+      "type": "boolean"
+    },
+    "componentStatementIDs": {
+      "title": "Component statement IDs",
+      "description": "The identifiers of all statements that are components of this statement.",
```

Comment:
Suggested description change:

"The identifiers of all component statements that provide detail about the indirect relationship between this Statement's `subject` and `interestedParty`. If this Ownership-or-control Statement has components, it MUST itself have a `componentOfStatement` value of 'false'."